src/test/java/com/iota/iri/service/APITest.java

```diff
@@ -0,0 +1,56 @@
+package com.iota.iri.service;
+
+import com.iota.iri.TransactionValidator;
+import com.iota.iri.conf.IotaConfig;
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.service.snapshot.SnapshotProvider;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Answers;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Collections;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.longThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Hmm, now that you have upgraded Mockito this doesn't give a warning.

I agree that it is nicer than then using `@rule`, the problem is that we started using rules elsewhere in the code.
@kwek20, what do you think? You want to make the switch?

Reply:
Did this out of habit. Just tell me what you prefer an I will change it. Another option would be not to use the runner at all and call `MockitoAnnotations.initMocks(this);` in the `@Before`. That way the tests stay open for other Runners.

Reply:
I think using the `@Rule` is fine for now. If we ever decide to make a change for Junit 5, we will need to do an overhaul anyways, and we can decide then.

If you want to change to a different runner, `@RunWith` is available.
Please change it to the following:

```
@Rule
public MockitoRule mockitoRule = MockitoJUnit.rule();
```

Reply:
@kwek20 OK, will do!